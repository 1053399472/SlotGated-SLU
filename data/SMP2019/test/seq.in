黑 米 红 豆 粥 的 做 法 。
湖 南 卫 视 娱 乐 节 目 。
怎 么 去 泉 城 广 场
中 央 台 五 套
到 北 京 的 火 车 班 次
上 海 到 合 肥 怎 么 坐 汽 车 ？
请 帮 我 打 开 电 影
哦 ， 听 新 闻 。
给 万 结 打 电 话
睡 觉 用 英 文 怎 么 说
双 色 球 的 开 奖 号 码 是 多 少 ？
帮 我 转 发 邮 件
昨 天 晚 上 湖 南 卫 视 演 什 么
番 红 烧 肉 的 做 法 。
松 江 钢 材 城 在 哪 里
可 乐 鸡 翅 怎 么 做 呀 晚 点 告 诉 我 。
读 书 频 道 上 个 星 期 五 有 播 放 新 闻 联 播 吗
浙 江 台
冰 糖 雪 梨
西 兰 花 怎 么 做 ？
美 菱 电 器 的 昨 日 收 盘 价 是 多 少
恶 熊 出 没
央 视 少 儿 频 道 的 节 目 的
我 想 听 音 乐 忘 情 水 。
打 电 话 给 王 安 娜
中 国 合 伙 人
去 汽 车 东 站 怎 么 走
青 海 卫 视 频 道 的 节 目 单
念 一 首 唐 诗 给 我 听 。
把 张 玉 娟 的 手 机 号 码 发 送 给 吴 伟
查 询 恒 光 电 影 院 即 将 上 映 的 电 影
安 阳 到 郑 州 的 火 车
杜 甫 兵 车 行 。
南 京 火 车 站 到 南 京 农 业 大 学
糖 醋 藕 怎 么 做 ？
打 开 u c 二 哦
中 超 赛 事 预 告 吗 ？
红 烧 肉 怎 么 做 ？ 嗯 ？
读 一 首 毛 泽 东 的 诗 。
需 要 你 帮 我 推 荐 一 下 电 影
银 耳 莲 子 羹 。
说 话 ， 给 我 搜 一 下 做 苹 果 麦 片 粥 是 怎 么 做 啊 ？
查 一 下 明 天 青 岛 飞 往 上 海 的 飞 机 都 有 哪 一 些 都 是 几 点 的
中 超 赛 事 预 告 ， 你 。
张 艺 谋 的 甄 嬛 传
找 湖 南 卫 视
帮 我 查 一 下 宁 波 到 郑 州 的 火 车
江 苏 阳 光 的 股 票 怎 么 样 了
我 想 听 歌 曲 传 奇
现 在 湖 南 卫 视 放 什 么 电 视 ？
发 信 息 给 健 美
上 美 空 网
我 要 看 免 费 的 小 说 。
最 近 有 什 么 好 看 的 电 视 连 续 剧 武 打 的 。
发 信 息 给 陈 明 哲
打 开 腾 讯 头 条 新 闻
红 烧 带 鱼 怎 么 做 ？ 你 ？
把 侨 天 慧 的 号 码 发 给 副 卡
怎 样 做 糖 醋 排 骨 时 ， 去 吃 饭 洗 澡 了 。
晋 江 到 温 州 的 火 车 票
7 月 2 5 号 去 上 海 的 航 班 。
随 便 给 我 放 几 首 歌
夜 访 吸 血 鬼
请 打 开 网 页
十 月 四 号 从 广 州 到 北 京 的 飞 机 票 多 少 钱
腹 泻 。
灌 阳 县 位 置
查 查 最 近 一 周 C C T V - 4 美 洲 有 放 我 爱 记 歌 词 么
去 金 湖 县 城
现 在 有 什 么 好 看 的 电 视 剧 ？
来 做 烧 卖 。
远 教 四 台
古 诗 长 恨 歌 。
四 喜 丸 子 怎 么 做 ？
六 安 市 飞 云 卫 生 院 在 哪 里
酸 菜 鱼 怎 么 做 啊
我 要 全 部 回 复
贵 溪 至 泉 州 的 火 车 票
我 要 回 复 这 个 邮 件
济 南 到 汉 中 的 飞 机 票
给 秦 丽 香 发 信 息
我 想 要 看 很 火 的 电 影
查 下 温 州 到 杭 州 的 火 车 票
怎 么 样 ？ 做 红 烧 肉 。
看 一 下 中 式 传 媒 股 票 的 涨 跌 幅
呼 叫 小 惠
我 要 听 张 雨 生 的 歌 。
回 放 昨 晚 的 焦 点 访 谈
我 想 听 伤 感 歌 曲
感 冒 注 意 什 么 ？
搜 索 泰 亚 股 份 股 票
你 不 来 一 首 涛 声 依 旧 ， 那 你 。
做 红 烧 肉 要 哪 些 材 料 呀 ！
翻 译 慷 慨 激 昂
查 询 下 北 京 首 创 的 股 票
帮 我 翻 译 一 下 我 饿 了
我 想 看 变 形 金 刚 三
钓 鱼 岛 新 闻
怎 么 治 疗 牛 皮 癣
独 立 纵 队 电 视 剧
做 面 包 的 配 方 。
来 时 打 来 首 唐 诗 绝 句 。
但 我 想 点 新 闻 。
今 天 有 什 么 新 闻 呢 。
宁 波 到 杭 州 的 汽 车
素 鸡
给 我 背 一 首 唐 诗 李 白 写 的
给 黄 杠 发 短 信
自 演 艺 ， 羊 肉 怎 么 做 ？
搜 索 七 乐 彩 的 公 示 号 码
我 想 听 春 天 里 这 首 歌
给 张 张 打 电 话
完 毕 用 英 文 怎 么 说
我 不 知 道 的 英 文 怎 么 说
看 千 金 归 来 第 五 集
我 要 看 校 园 小 说 致 我 们 终 将 逝 去 的 青 春
豆 豉 鲮 鱼 油 麦 菜
一 千 五 英 语 怎 么 说
姐 你 红 烧 肉 怎 么 做 呀
上 海 宝 钢 股 票
给 岚 岚 发 短 信
保 险 ， 酸 辣 土 豆 丝 怎 么 炒 ？
请 给 我 打 一 个 谜 语 。
封 神 太 子
帮 我 看 一 下 糖 醋 排 骨 怎 么 烧 ？
打 开 u c 浏 览 器 v 八
打 开 音 乐 播 放 器
查 询 东 吴 证 券 的 股 票
翻 译 光 大 银 行
甲 型 肝 炎 。
尽 在 不 言 中 下 一 句 是 什 么 ？
洛 阳 至 西 安 的 火 车
明 天 去 南 昌 的 航 班 。
类 风 湿 怎 么 治 ？
合 肥 的 大 润 发 在 哪
小 鸡 炖 蘑 菇 的 做 法 爸
听 江 苏 交 通 广 播 网
我 说 白 鱼 怎 么 做
陈 周 的 电 话 号 码 是 多 少
鼻 息 肉
哦 ， 中 超 比 赛 时 间 三 比 五 。
体 育 赛 事 直 播
找 一 首 游 鸿 明 的 音 乐
2 0 1 2 年 美 国 公 告 牌 音 乐 大 奖 颁 奖 礼
后 天 新 闻
葫 芦 鸡 怎 么 做 ？
帮 我 找 到 去 安 徽 黄 山 的 路
咱 们 结 婚 吧 什 么 时 候 播 出
查 看 短 消 息
让 我 猜 个 谜 语 。
明 天 南 京 到 徐 州 的 火 车
查 询 明 天 南 阳 到 上 海 的 航 班
请 帮 我 调 频 8 9 . 6 武 汉 交 通 广 播 电 台
去 汽 车 南 站 怎 么 走
发 信 息 给 熊 伟
还 路 阻 少 茶 汤 的 做 法 。
背 将 进 酒 。
播 放 郁 可 唯 的 歌
烟 台 到 齐 齐 哈 尔 的 火 车 票
你 们 什 么 搞 笑 剧
把 李 志 强 的 号 码 发 给 贾 洪 鉴
给 王 帅 发 个 短 信 说 他 让 他 起 来 尿 尿
打 开 4 3 9 9 小 游 戏
我 要 写 条 新 邮 件
给 我 查 一 下 最 近 在 播 哪 些 电 影
去 枣 庄 的 火 车
给 张 金 波 发 短 消 息 今 天 中 午 一 起 吃 饭
进 入 讯 飞 官 网
板 栗 怎 么 煮 ？
查 询 后 天 广 州 到 上 海 的 航 班
我 要 看 现 代 重 生 小 说
蒜 茄 子 怎 么 做 ？
打 开 互 联 网 啦
帮 我 查 一 下 江 西 铜 业 股 票
哪 个 电 视 台 在 演 叶 问 。
回 锅 肉 。
青 瓷 电 视 剧
不 懂 饭 的 做 法 。
给 晓 琴 发 短 信 你 在 干 吗
查 寻 标 兵 的 号 码
C C T V - 5 节 目 表
看 中 央 七 频 道 讲 武 堂 节 目
来 一 首 李 煜 的 词 。
发 条 短 信 给 张 亚 东
一 个 鬼 子 都 不 留
看 一 看 七 乐 彩 的 信 息
呼 叫 徐 媛
刘 林 面 怎 么 做 啊 ？
查 寻 来 宾 到 柳 州 的 火 车
打 电 话 给 张 志 国
找 愤 怒 的 小 鸟
如 何 做 西 红 柿 炒 蛋 。
调 到 新 疆 综 艺
现 在 中 央 一 台 在 播 什 么 节 目 ？
打 开 德 州 扑 克
苏 宁 易 购
我 想 写 一 个 新 邮 件
萝 卜 炖 排 骨 汤 怎 么 做 ？
我 要 听 流 行 歌 曲
明 天 去 纽 约 的 航 班 。
明 天 温 州 到 上 海 的 航 班 。
腿 英 文 怎 么 说
下 载 汉 化 软 件
从 这 里 到 市 中 心 怎 么 走
我 想 看 足 球 中 超 ， 第 2 5 轮 的 比 赛 时 间 。
贵 州 卫 视 今 晚 看 什 么 电 视 剧 ？
查 一 下 杭 州 到 鹰 潭 的 火 车 票
我 想 打 电 话 给 李 四
打 开 网 页 猫 扑
破 伤 风 该 怎 么 处 理
南 京 到 厦 门 的 火 车 票
天 才 碰 麻 瓜
李 白 的 将 进 酒 。
我 要 看 图 文 小 说
鸡 蛋 怎 么 做 ？
最 新 的 新 闻 播 报 一 下
八 月 十 号 从 重 庆 到 武 汉 的 航 班
明 天 长 沙 到 上 海 的 航 班
从 我 这 里 去 芙 蓉 区 政 府 怎 么 去
聚 友 网
姨 ， 最 近 有 什 么 新 闻 。
读 一 首 古 诗 给 我 听 。
猜 谜 语 ， 听 了 。
南 京 到 成 都 的 机 票
炝 拌 土 豆 丝 。
我 要 买 彩 票 。
打 一 个 谜 语 男 人 的 腿 残 。
那 烤 箱 做 面 包 怎 么 做 ？
中 国 太 保 股 票
沙 眼 。
帮 我 找 出 红 烧 肉 的 做 法 。
广 州 到 南 宁 的 火 车
煮 水 饺 。
说 话 的 英 文 怎 么 写
呼 叫 琳 琳
开 微 信
瘦 肉 粥 。
请 帮 我 调 频 1 0 6 . 1 长 沙 音 乐 广 播 电 台
从 北 海 到 东 莞 的 汽 车 。
我 叫 小 沈 阳 这 首 歌
宿 州 到 北 京 的 火 车 有 哪 些
羊 肉 怎 么 烧 ？
发 烧 。
妖 精 的 尾 巴
小 杰 炖 蘑 菇 怎 么 做 ？
高 清 广 东 台
凉 菜 是 怎 么 做 的 ？
捍 卫 者
拌 面 该 怎 么 做 ？
你 还 没 有 结 婚 用 英 语 怎 么 讲
广 州 至 南 宁 航 班 。
打 电 话 麦 亚 伦
土 豆 粉 怎 么 做 好 吃 ？
做 红 酒 怎 么 做 啊 ？
来 一 首 我 们 的 歌 谣 。
打 开 u c 浏 览 器 登 录 新 浪 网 主 页
影 视 流 星 花 园
给 我 背 一 首 诗 。
来 首 歌 吗 ？
海 南 旅 游 卫 视
开 启 q q
我 看 呀 中 央 五 台
再 猜 一 个 谜 语 。
广 州 恒 大 的 赛 程 。
九 江 到 景 德 镇 的 汽 车 。
查 一 下 济 南 到 成 都 的 航 班
给 栾 佳 佳 发 短 信 说 回 到 济 南 了
我 回 去 看 昨 天 晚 上 播 出 的 爸 爸 去 哪 儿 3 1 频 道
皮 冻 的 做 法 。
白 灼 基 尾 虾 的 做 法 。
