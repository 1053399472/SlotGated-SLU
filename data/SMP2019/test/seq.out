B-dishName I-dishName I-dishName I-dishName E-dishName O O O O
B-tvchannel I-tvchannel I-tvchannel E-tvchannel B-category E-category O O O
O O O B-endLoc_poi I-endLoc_poi I-endLoc_poi E-endLoc_poi
B-name I-name I-name I-name E-name
O B-endLoc_city E-endLoc_city O O O O O
B-Src E-Src O B-Dest E-Dest O O O O O O
O O O O O B-category E-category
O O O O O O
O B-name E-name O O O
B-content E-content O B-target E-target O O O
B-name I-name E-name O O O O O O O O O
O O O O O O
B-datetime_date E-datetime_date B-datetime_time E-datetime_time B-tvchannel I-tvchannel I-tvchannel E-tvchannel O O O
O B-dishName I-dishName E-dishName O O O O
B-location_poi I-location_poi I-location_poi I-location_poi E-location_poi O O O
B-dishName I-dishName I-dishName E-dishName O O O O O O O O O O
B-tvchannel I-tvchannel I-tvchannel E-tvchannel B-datetime_date I-datetime_date I-datetime_date I-datetime_date E-datetime_date O O O B-name I-name I-name E-name O
B-name I-name E-name
B-dishName I-dishName I-dishName E-dishName
B-ingredient I-ingredient E-ingredient O O O O
B-name I-name I-name E-name O B-yesterday E-yesterday O O O O O O
O B-name I-name E-name
B-tvchannel I-tvchannel I-tvchannel I-tvchannel I-tvchannel E-tvchannel O O O O
O O O O O B-song I-song E-song O
O O O O B-name I-name E-name
B-name I-name I-name I-name E-name
O B-endLoc_poi I-endLoc_poi I-endLoc_poi E-endLoc_poi O O O
B-tvchannel I-tvchannel I-tvchannel E-tvchannel O O O O O O
O O O S-dynasty O O O O O
O B-name I-name E-name O B-category E-category O O O O O B-receiver E-receiver
O O B-theatre I-theatre I-theatre I-theatre E-theatre O O O O O O O
B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
B-author E-author B-name I-name E-name O
O O B-startLoc_poi I-startLoc_poi E-startLoc_poi B-startLoc_city E-startLoc_city I-endLoc_poi I-endLoc_poi I-endLoc_poi I-endLoc_poi E-endLoc_poi
B-dishName I-dishName E-dishName O O O O
O O B-name E-name O O
B-category E-category O O O O O O
B-dishName I-dishName E-dishName O O O O O O
O O O B-author I-author E-author O O O
O O O O O O O O O B-category E-category
B-dishName I-dishName I-dishName I-dishName E-dishName O
O O O O O O O O O B-dishName I-dishName I-dishName I-dishName E-dishName O O O O O O
O O O B-startDate_date E-startDate_date B-startLoc_city E-startLoc_city O O B-endLoc_city E-endLoc_city O O O O O O O O O O O O O
B-category E-category O O B-type E-type O O O
B-artist I-artist E-artist O B-name I-name E-name
O B-name I-name I-name E-name
O O O O O B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
B-name I-name I-name E-name O O O O O O O
O O O O O B-song E-song
B-datetime_time E-datetime_time B-tvchannel I-tvchannel I-tvchannel E-tvchannel O O O O O O
O O O O B-name E-name
O B-name I-name E-name
O O O O O O O O O
B-timeDescr E-timeDescr O O O B-scoreDescr E-scoreDescr O B-category I-category I-category I-category E-category B-tag E-tag O O
O O O O B-name I-name E-name
O O B-media E-media B-category E-category O O
B-dishName I-dishName I-dishName E-dishName O O O O O O
O B-name I-name E-name O O O O O B-receiver E-receiver
O O O B-dishName I-dishName I-dishName E-dishName O O O O O O O O O
B-startLoc_area E-startLoc_area O B-endLoc_city E-endLoc_city O O O O
B-startDate_date I-startDate_date I-startDate_date I-startDate_date E-startDate_date O B-endLoc_city E-endLoc_city O O O O
O O O O O O O O
B-name I-name I-name I-name E-name
O O O O O
B-startDate_date I-startDate_date I-startDate_date E-startDate_date O B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O O O O O
B-keyword E-keyword O
B-location_area I-location_area E-location_area O O
O O B-datetime_date I-datetime_date I-datetime_date E-datetime_date B-tvchannel I-tvchannel I-tvchannel I-tvchannel I-tvchannel I-tvchannel I-tvchannel E-tvchannel O O B-name I-name I-name I-name E-name O
O B-endLoc_poi I-endLoc_poi I-endLoc_poi E-endLoc_poi
O O O O O B-scoreDescr E-scoreDescr O B-category I-category E-category O
O O B-dishName E-dishName O
B-name I-name I-name E-name
O O B-name I-name E-name O
B-dishName I-dishName I-dishName E-dishName O O O O
B-location_city I-location_city E-location_city B-location_poi I-location_poi I-location_poi I-location_poi E-location_poi O O O
B-dishName I-dishName E-dishName O O O O
O O O O O O
B-startLoc_area E-startLoc_area O B-endLoc_city E-endLoc_city O O O O
O O O O O O O O
B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O O
O B-name I-name E-name O O O
O O O O O S-popularity O B-category E-category
O O B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O O
O O O O O B-dishName I-dishName E-dishName O
O O O B-name I-name I-name E-name O O O O O O
O O B-name E-name
O O O B-artist I-artist E-artist O O O
O O B-datetime_time E-datetime_time O B-name I-name I-name E-name
O O O B-category E-category O O
B-keyword E-keyword O O O O O
O O B-name I-name I-name E-name O O
O O O O O B-song I-song I-song E-song O O O O
O B-dishName I-dishName E-dishName O O O O O O O
O O B-content I-content I-content E-content
O O O B-name I-name I-name E-name O O O
O O O O O O B-content I-content E-content
O O O B-name I-name I-name I-name E-name
B-keyword I-keyword E-keyword O O
O O O O B-keyword I-keyword E-keyword
B-name I-name I-name E-name B-category I-category E-category
O B-dishName E-dishName O O O O
O O O O O S-dynasty O B-name E-name O
O O O O O O O
B-datetime_date E-datetime_date O O O O O O O
B-Src E-Src O B-Dest E-Dest O O O
B-dishName E-dishName
O O O O O S-dynasty O B-author E-author O O
O B-name E-name O O O
O O O O B-ingredient E-ingredient O O O O
O O B-name I-name E-name O O O O O
O O O B-song I-song E-song O O O
O B-name E-name O O O
B-content E-content O B-target E-target O O O
B-content I-content I-content E-content O B-target E-target O O O
O B-name I-name I-name E-name B-episode I-episode E-episode
O O O B-category E-category O O B-name I-name I-name I-name I-name I-name I-name I-name I-name E-name
B-dishName I-dishName I-dishName I-dishName I-dishName I-dishName E-dishName
B-content I-content E-content B-target E-target O O O
O O B-dishName I-dishName E-dishName O O O O
O O B-name E-name O O
O B-name E-name O O O
O O O B-dishName I-dishName I-dishName I-dishName E-dishName O O O O
O O O O O O B-category E-category O
B-name I-name I-name E-name
O O O O O B-dishName I-dishName I-dishName E-dishName O O O O
O O B-name I-name I-name I-name I-name I-name E-name
O O B-name I-name I-name I-name E-name
O O B-name I-name I-name E-name O O O
O O B-content I-content I-content E-content
B-keyword I-keyword I-keyword E-keyword O
B-keyword I-keyword I-keyword I-keyword E-keyword B-queryField I-queryField E-queryField O O O O
B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
B-startDate_date E-startDate_date O B-endLoc_city E-endLoc_city O O O O
B-keyword I-keyword E-keyword O O O O
B-location_city E-location_city O B-location_poi I-location_poi E-location_poi O O
B-dishName I-dishName I-dishName I-dishName E-dishName O O O O
O B-name I-name I-name I-name I-name E-name O
O O B-ingredient E-ingredient O O O
B-name E-name O O O O O O O O
B-keyword I-keyword E-keyword
O O B-category E-category O O O O O O O O
B-name I-name I-name E-name O O
O O O B-artist I-artist E-artist O O O
B-datetime_date I-datetime_date I-datetime_date I-datetime_date E-datetime_date B-name I-name I-name I-name I-name I-name I-name I-name I-name I-name I-name E-name
B-datetime_date E-datetime_date O O
B-dishName I-dishName E-dishName O O O O
O O O O O B-endLoc_province E-endLoc_province B-endLoc_poi E-endLoc_poi O O
B-name I-name I-name I-name E-name O O O O O O
O O O O O
O O O O B-category E-category O
B-startDate_date E-startDate_date B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
O O B-startDate_date E-startDate_date B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
O O O O O B-code I-code I-code E-code B-name I-name I-name I-name I-name I-name I-name E-name
O B-endLoc_poi I-endLoc_poi I-endLoc_poi E-endLoc_poi O O O
O O O O B-name E-name
O O O O B-dishName E-dishName O O O O
O B-name I-name E-name O
O O B-artist I-artist E-artist O O
B-startLoc_city E-startLoc_city O B-endLoc_city I-endLoc_city I-endLoc_city E-endLoc_city O O O O
O O O O B-category I-category E-category
O B-name I-name E-name O O O O O B-receiver I-receiver E-receiver
O B-name E-name O O O O O O B-content I-content I-content I-content I-content E-content
O O B-name I-name I-name I-name I-name I-name E-name
O O O O O O O
O O O O O O O O O O O O O
O B-endLoc_city E-endLoc_city O O O
O B-name I-name E-name O O O O B-content I-content I-content I-content I-content I-content I-content E-content
O O B-name I-name I-name E-name
B-ingredient E-ingredient O O O O
O O B-startDate_date E-startDate_date B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
O O O B-category I-category I-category E-category O O
B-dishName I-dishName E-dishName O O O O
O O O O O O
O O O O O B-name I-name I-name E-name O O
O O O O O O O B-name E-name O
B-dishName I-dishName E-dishName O
B-name E-name B-category I-category E-category
O O S-dishName O O O O
O B-name E-name O O O B-content I-content I-content E-content
O B-name I-name E-name O O O
B-tvchannel I-tvchannel I-tvchannel I-tvchannel I-tvchannel E-tvchannel O O O
O B-tvchannel I-tvchannel I-tvchannel I-tvchannel E-tvchannel B-name I-name E-name O O
O O O B-author E-author O O O
O O O O O B-name I-name E-name
B-name I-name I-name I-name I-name I-name E-name
O O O B-name I-name E-name O O O
O O B-name E-name
O O S-ingredient O O O O O
O O B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
O O O O B-name I-name E-name
O B-name I-name I-name I-name E-name
O O O B-dishName I-dishName I-dishName I-dishName E-dishName O
O O B-name I-name I-name E-name
B-datetime_time E-datetime_time B-tvchannel I-tvchannel I-tvchannel E-tvchannel O O O O O O O
O O B-name I-name I-name E-name
B-name I-name I-name E-name
O O O O O O O O
B-dishName I-dishName I-dishName I-dishName I-dishName E-dishName O O O O
O O O B-category E-category O O
B-startDate_date E-startDate_date O B-endLoc_city E-endLoc_city O O O O
B-startDate_date E-startDate_date B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O O
S-content B-target E-target O O O
O O B-name E-name O O
O O O O B-endLoc_poi I-endLoc_poi E-endLoc_poi O O O
O O O O O B-category E-category O O O O O O O O O O O
B-tvchannel I-tvchannel I-tvchannel E-tvchannel B-datetime_time E-datetime_time O O O B-category I-category E-category O
O O O B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O O
O O O O O O B-name E-name
O O O O B-name E-name
B-keyword I-keyword E-keyword O O O O O
B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O O
B-name I-name I-name I-name E-name
B-author E-author O B-name I-name E-name O
O O O B-category E-category O O
B-ingredient E-ingredient O O O O
B-keyword E-keyword O O O O O O O
B-startDate_date I-startDate_date I-startDate_date E-startDate_date O B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
B-startDate_date E-startDate_date B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
O O O O O B-endLoc_poi I-endLoc_poi I-endLoc_poi I-endLoc_poi E-endLoc_poi O O O
B-name I-name E-name
O O B-datetime_date E-datetime_date O O O O O O
O O O O O O O O O
O B-category E-category O O O O
B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
B-dishName I-dishName I-dishName I-dishName E-dishName O
O O O O O O
O O O B-category E-category O O O O O O
O O O O B-dishName E-dishName O O O O
B-name I-name I-name E-name O O
B-keyword E-keyword O
O O O O B-dishName I-dishName E-dishName O O O O
B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
O B-dishName E-dishName O
B-content E-content O B-target E-target O O O
O O B-name E-name
O B-name E-name
B-dishName I-dishName E-dishName O
O O O O O B-code I-code I-code I-code E-code B-name I-name I-name I-name I-name I-name I-name E-name
O B-Src E-Src O B-Dest E-Dest O O O O
B-song I-song I-song I-song E-song O O O
B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O O O O
B-ingredient E-ingredient O O O O
B-keyword E-keyword O
B-name I-name I-name I-name E-name
O O B-dishName I-dishName E-dishName O O O O
B-resolution E-resolution B-name I-name E-name
B-keyword E-keyword O O O O O O
B-name I-name E-name
B-dishName E-dishName O O O O O
B-content I-content I-content I-content I-content E-content O B-target E-target O O O
B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
O O O B-name I-name E-name
B-ingredient I-ingredient E-ingredient O O O O O O
O B-ingredient E-ingredient O O O O O
O O O B-song I-song I-song I-song E-song O
O O B-name I-name I-name I-name E-name O O O O O O O
B-category E-category B-name I-name I-name E-name
O O O O O O O
O O O O O
O O B-name I-name I-name E-name
O O B-name E-name
O O O B-name I-name I-name E-name
O O O O B-category E-category O
B-name I-name I-name E-name O O O O
B-Src E-Src O B-Dest I-Dest E-Dest O O O O
O O O B-startLoc_city E-startLoc_city O B-endLoc_city E-endLoc_city O O O
O B-name I-name E-name O O O O B-content I-content I-content I-content E-content
O O O O B-datetime_date E-datetime_date B-datetime_time E-datetime_time O O O B-name I-name I-name I-name E-name B-code E-code O O
B-ingredient E-ingredient O O O O
B-dishName I-dishName I-dishName I-dishName E-dishName O O O O
